release-notes/versioned/pulsar-3.0.0.md

```diff
@@ -0,0 +1,484 @@
+- [Proxy] Allow proxy to pass same role for authRole and originalRole by @michaeljmarshall in [#19557](https://github.com/apache/pulsar/pull/19557)
+- [Proxy] Fix using wrong client version in pulsar proxy by @RobertIndie in [#19540](https://github.com/apache/pulsar/pull/19540)
+- [Proxy] Fix JKS TLS transport by @nodece in [#19485](https://github.com/apache/pulsar/pull/19485)
+- [Proxy] Modify check waitingForPingResponse with volatile (#12615)" by @michaeljmarshall in [#19439](https://github.com/apache/pulsar/pull/19439)
+- [Proxy] Only go to connecting state once by @michaeljmarshall in [#19331](https://github.com/apache/pulsar/pull/19331)
+- [Proxy] PIP 97: Implement for ProxyConnection by @michaeljmarshall in [#19292](https://github.com/apache/pulsar/pull/19292)
+- [Proxy] Remove unused AuthenticationDataSource variable by @michaeljmarshall in [#19278](https://github.com/apache/pulsar/pull/19278)
+- [Proxy] Use ChannelVoidPromise to avoid useless promise objects creation by @nicoloboschi in [#19141](https://github.com/apache/pulsar/pull/19141)
+- [Proxy] Prevent DNS reverse lookup when physical address is an IP address by @lhotari in [#19028](https://github.com/apache/pulsar/pull/19028)
+- [Proxy] Fix logging in shutdown when proxy shutdown by @coderzc in [#18622](https://github.com/apache/pulsar/pull/18622)
+- [Proxy] Fix refresh client auth by @nodece in [#17831](https://github.com/apache/pulsar/pull/17831)
+- [Proxy] Update proxy lookup throw exception type by @tuteng in [#17600](https://github.com/apache/pulsar/pull/17600)
+- [Proxy] Fix producer info logging fails when proxyLogLevel > 0 by @zengguan in [#17175](https://github.com/apache/pulsar/pull/17175)
+- [Proxy] Support proxy limit maximum connections per IP by @mattisonchao in [#17167](https://github.com/apache/pulsar/pull/17167)
+- [Proxy] Remove empty else code block when initialize setMetadataStoreUrl  by @aloyszhang in [#17122](https://github.com/apache/pulsar/pull/17122)
+- [Proxy] Use correct address for HAProxyMessage destination by @michaeljmarshall in [#16045](https://github.com/apache/pulsar/pull/16045)
+- [Schema] Fix delete schema command doc by @congbobo184 in [#19411](https://github.com/apache/pulsar/pull/19411)
+- [Schema] Fix schema get value doc problem by @congbobo184 in [#17740](https://github.com/apache/pulsar/pull/17740)
+- [Schema] Fix producer/consume permission can’t get v1/schema by @yuruguo in [#16018](https://github.com/apache/pulsar/pull/16018)
+
+
+### Library updates
+
+- Upgrade the BookKeeper dependency to 4.16.1 by @hangc0276 in [#20127](https://github.com/apache/pulsar/pull/20127)
+- Upgrade snakeyaml version to 2.0 by @Technoboy- in [#20085](https://github.com/apache/pulsar/pull/20085)
+- Upgrade the BookKeeper dependency to 4.16.0 by @hangc0276 in [#20049](https://github.com/apache/pulsar/pull/20049)
+- Upgrade bookkeeper to 4.15.4 by @zymap in [#19812](https://github.com/apache/pulsar/pull/19812)
```

Comment:
I wonder if it would be better to omit this since it has been superseded by https://github.com/apache/pulsar/pull/20127

I think it would be much clearer for users because they only care of the actual version present in 3.0.0 and not all the process that has happened during the development

this also applies to other dependencies

Reply:
Thanks. Fixed them in https://github.com/apache/pulsar-site/pull/545/commits/71951a2f6615ad9616dac272729808ffbcbbf337